### ISSUE TITLE
Multi-Monitor Macs

### ISSUE THREAD

**Issue by PetieM** [Ylianst/MeshCentral#3773]
Hey all, just popping in to follow up on #1757 regarding Macs with multiple monitors. Has there been any movement in making this work? I understand and appreciate that the Mac agent is not as high a priority as the Windows one and that there are limited development resources to put towards it but if it's possible, being able to switch screens on a Mac (or even see all of them, in the short-term - right now, you can only see the one designated as primary) would be huge. Thanks!

**Comment by krayon007** [comment]
Not yet. My problem was that my secondary monitors were display link monitors, which isn't supported by macos. I got new monitors a little bit ago, so I can look into it.

**Comment by PetieM** [comment]
Fair enough, thanks for the update! Happy to guinea pig as well if having a second system to test with would help.

**Comment by JSuenram** [comment]
+1

**Comment by matt-cylanda** [comment]
+1

**Comment by vitobotta** [comment]
Just ran into this. Is there any workaround?

**Comment by vitobotta** [comment]
Any update on this? Thanks

**Comment by SinisterSpatula** [comment]
Hello I noticed this issue when I tried mesh central for the first time today.  For now I am able to use "Web-vnc" option instead and it connects to the correct display.  Is it possible for me to configure the mac agent to always use the other monitor (the one that Web-vnc displays is the one I want to use).

**Comment by si458** [comment]
hmmm I dont no if the web-vnc supports multiple monitors? Never tried? Will have a look later!
I know for a fact the meshagent sadly on mac doesn't support multiple monitors
